govuk-one-login/authentication-tech-docs
# fix heading in changelog and remove drawio temp file

## Why

Documentation heading is not generating a correct link
![image](https://github.com/govuk-one-login/authentication-tech-docs/assets/34273121/f6d0f651-bd34-49a2-b899-e4d73d8ef259)

## What

- Replace `<h2>` heading in _changelog with `##` to generate correct heading link
- Remove temporary drawio *.bkp file

## Technical writer support

No

## How to review

- pull the branch 
- `./preview-with-docker.sh`
- ensure http://localhost:4567/#documentation-updates works

## Changelog

If this change is significant (for example, launching a new feature or deprecating a feature), you should update the changelog found at `partials/_changelog.erb`  under the heading 'Documentation updates'.

## Confirm
